docsrc/overview/who_is_cyrus.rst

```diff
@@ -1,21 +1,20 @@
 Who Is Cyrus
 ============
 
-The Cyrus project was originally started at `Carnegie Mellon University`_ but it has since attracted a very dedicated and diverse group of core contributors. Carnegie Mellon University remains active in development of Cyrus and also provides the infrastructure on which cyrusimap.org runs today, but CMU is now just one of many core contributors to Project Cyrus.
+The Cyrus project was originally started at `Carnegie Mellon University`_ but it has since attracted a very dedicated and diverse group of core contributors.
 
 Core Contributors
 -----------------
 These are the core organizations who are providing dedicated resources to the Cyrus Project.
 
-**FastMail**
+**Fastmail**
 
-FastMail_ is a leading e-mail hosting provider that runs on Cyrus software.
+Fastmail_ is a leading e-mail hosting provider that runs on Cyrus software.
 
 *	Bron Gondwana
-*   Ellie Timoney
-*   Conrad Kleinespel
-*   Chris Davies
```

Comment:
There's some kinda whitespace mismatch here, does the Bron line have a tab in it?

Reply:
Scrolling further, the same comment applies to most/all the other bulleted lists.  I'd like to see it to be consistent, and I don't like the hard tab so I would lean toward replacing the old tabs rather than making the new lines match the old

Reply:
I've replaced all the tabs with single spaces to match the majority of bullets.